authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,186 @@
+
+Both modes of communication - crosslinks and ICMP - are asynchronous and thus non-instant. The appearance of real-time communication will be a matter of userland implementation.
+
+## Node types
+
+In Ethereum, the following economy-user and node roles (client software types) exist:
+
+|Name       |Role                                                  |Incentivized|Optional|
+|-----------|------------------------------------------------------|------------|--------|
+|Beacon node|Full node of all shards. Passes on data to validators.|No          |No      |
+|Validator  |Proposes and attests to blocks based on beacon data.  |Yes         |No      |
+|VDF ASIC   |Enhances RANDAO to provide better randomness.         |No          |Yes     |
+
+Beacon nodes can potentially be incentivized by being archive nodes of all shards, which means they keep even the data which was wiped due to state rent expiration. A state-rent wiped contract can be restored by an archive node, for a fee. Archive nodes as-a-service are probable in Ethereum 2.0.
+
+In Polkadot:
+
+|Name       |Role                                                        |Incentivized|Optional |
+|-----------|------------------------------------------------------------|------------|---------|
+|Collator   |Full node of particular shard. Passes on data to validators.|Possibly    |No       |
+|Validator  |Full client of all parachains and parathreads. Proposes and attests to blocks based on collator data.      |Yes         |No       |
+|Nominator  |Nominates a validator through which to proxy-stake.         |Yes         |In theory|
+|Fishermen  |Monitor the network for misdeeds like equivocation or illegal randomness.|Yes|Yes|
+
+Validators can be fishermen, which makes fishermen optional. Nominators are _theoretically_ unnecessary, i.e. the network would work fine without them if the validators had enough self-stake to keep themselves in the validator pool.
+
+## Inflation
+
+In both Polkadot and Ethereum the inflation is dynamic and defined as "minimum necessary inflation". The exact number will depend on the amount of staked coins.
```

Comment:
Perhaps add estimates here for Eth 2.0 and Polkadot, even though they are of course estimates?  Just to give a general idea for comparison.

Reply:
Hmm, 20% is mentioned later, but I would think this would be a better place for it (just my opinion).